E3Next/Processors/Loot.cs

```diff
@@ -181,8 +181,8 @@ public static void Process()
             if (!E3.CharacterSettings.Misc_AutoLootEnabled) return;
             if(!Assist.IsAssisting)
             {
-
-                if (!Basics.InCombat() || E3.GeneralSettings.Loot_LootInCombat)
+                long currentTimestamp = Core.StopWatch.ElapsedMilliseconds;
```

Comment:
I thought about putting in a timestamp based on if Basics.InCombat changed from InCombat -> !InCombat. Or adding another customization for "loot if no more aggressive mobs within X range". But this seems good enough for now. Downside is if you are doing an assist and it takes longer to kill it (i.e. named), your auto looter will loot right after.

Could set a higher wait, but then looter is sitting there after combat ends. So probably needs another piece here eventually

Reply:
need to do this when the assist ENDS, not begins :)

Reply:
Ah, didn't realize there was an AssistOff() func I can use for this. Updated